config/default.yml

```diff
@@ -148,6 +148,71 @@ Style/BarePercentLiterals:
     - percent_q
     - bare_percent
 
+Style/Blocks:
+  EnforcedStyle: multiline
+  SupportedStyles:
+    # The `multiline` style enforces braces around single line blocks and
+    # do..end around multi-line blocks.
+    - multiline
```

Comment:
I don't like this name - after all the idea if to differentiate between single-line and multi-line blocks here.